ui/component/notificationContentChannelMenu/view.jsx

```diff
@@ -1,24 +1,23 @@
 // @flow
 import * as ICONS from 'constants/icons';
 import React from 'react';
-import { MenuItem } from '@reach/menu-button';
+import { MenuItem } from 'component/common/menu-components';
 import { parseURI } from 'util/lbryURI';
-import Icon from 'component/common/icon';
 
 type Props = {
   uri: string,
   notificationsDisabled: boolean,
```

Comment:
The button text seems to be stuck at the initial state, i.e. when toggled, the text doesn't change.

It seems to be happening in production as well, but not as frequent.  In this branch, it happens to all my New Content notifications.

<img width="265" alt="image" src="https://user-images.githubusercontent.com/64950861/153530437-958d1f55-d6ae-407f-a6aa-da6557c29607.png">

Reply:
Oh, never mind.  It does work sometime in this branch too.  Probably just waiting for the fetch to resolve back to redux or something.  Fix only if it's obvious what's going on.